src/tests/all-tests.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+var fluid = fluid || require('infusion');
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.smtp");
+require("../js/mailserver");
+
+var jqUnit = fluid.require("jqUnit");
+var fs     = require("fs");
+
+var nodemailer    = require("nodemailer");
+var smtpTransport = require('nodemailer-smtp-transport');
+var options = {
+    secure: false,
+    port:   4026,
+    ignoreTLS: true
+};
+var transporter   = nodemailer.createTransport(smtpTransport(options));
```

Comment:
Rather than being delivered literally via function calls, this crucial piece of implementation needs to be a configurable piece of component material that we can construct from configuration and wire to the server.